app/js/components/drugOrderEntry/index.jsx

```diff
@@ -35,6 +35,38 @@ export class SearchAndAddOrder extends React.PureComponent {
         </td>
       );
       this.handleEditActiveDrugOrder({ ...selectedOrder, status: activity }, details);
+    } else if (activity === 'DAFT_ORDER_EDIT' && selectedOrder) {
+      this.setState({
+        draftOrder: selectedOrder,
+        editDrugUuid: selectedOrder.drug,
+        editDrugName: selectedOrder.drugName,
+        orderNumber: selectedOrder.orderNumber,
+      });
+    }
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    const { selectedOrder, activity } = this.props;
+    if (activity === 'DAFT_ORDER_EDIT' && selectedOrder &&
```

Comment:
Is this 'DAFT_ORDER_EDIT' a typo or is it on purpose?

Reply:
Hi @Daramola98 
`DAFT_ORDER_EDIT` is actually a typo and surprisingly, I used it across all its location like that and that is why the functionality works 😄.
I would make sure to fix the typo.
Thanks!

Reply:
Great